deployments/helm/templates/deployment.yaml

```diff
@@ -58,10 +58,8 @@ spec:
             memory: {{ .Values.deployment.resources.requests.memory }}
         {{- if not .Values.backup.enabled }}
         args:
-        - --chain_id
-        - {{ .Values.starknet.chainId }}
-        - --central.url
-        - {{ .Values.starknet.centralUrl }}
+        - --config_file
+        - /app/config/presets/{{ .Values.starknet.preset }}
```

Comment:
[backup.sh](https://github.com/starkware-libs/papyrus/blob/main/deployments/helm/files/backup.sh) file needs to be modified also